frontend/lib/home/app_flows_stack.dart

```diff
@@ -42,8 +45,8 @@ class AppFlowsStack extends StatelessWidget {
   Widget _buildNavigatorForAppFlow(AppFlow appFlow) {
     return Navigator(
       key: appFlow.navigatorKey,
-      onGenerateRoute: (settings) => MaterialPageRoute(
-          settings: settings, builder: (context) => appFlow.widget),
+      onGenerateRoute: (settings) =>
+          AppRoute(settings: settings, builder: (context) => appFlow.widget),
```

Comment:
AppRoute adds a backgroundColor such that transitions between routes do not "glitch"